File: FacturadorLocal (2)/FacturadorEstaciones/FacturadorApi/Models/FacturaCanastilla.cs
﻿using FactoradorEstacionesModelo.Objetos;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EnviadorInformacionService.Models
{
    public class FacturaCanastilla
    {
		public int FacturasCanastillaId { get; set; }
		public DateTime fecha { get; set; }
		public Resolucion resolucion { get; set; }
		public int consecutivo { get; set; }
		public string estado { get; set; }
		public Tercero terceroId { get; set; }
		public int impresa { get; set; }
		public bool enviada { get; set; }
		public FormasPagos codigoFormaPago { get; set; }
		public List<CanastillaFactura> canastillas { get; set; }
		public float subtotal { get; set; }
		public float descuento { get; set; }
		public float iva { get; set; }
		public float total { get; set; }
	}
}

File: FacturadorSIGES/app/src/main/java/Modelo/FacturaCanastilla.java
package Modelo;

import java.util.Date;
import java.util.List;

public class FacturaCanastilla {
    public int FacturasCanastillaId;
    public Date fecha;
    public Resolucion resolucion ;
    public int consecutivo;
    public String estado;
    public Tercero terceroId ;
    public int impresa ;
    public int enviada ;
    public FormasPagos codigoFormaPago ;
    public List<CanastillaFactura> canastillas ;
    public float subtotal ;
    public float descuento ;
    public float iva;
    public float total;
}
